test/csuite/wt8246_compact_rts_data_correctness/main.c

```diff
@@ -91,39 +85,29 @@ int
 main(int argc, char *argv[])
 {
     TEST_OPTS *opts, _opts;
+    int i, j;
 
     opts = &_opts;
     memset(opts, 0, sizeof(*opts));
     testutil_check(testutil_parse_opts(argc, argv, opts));
 
-    testutil_assert(run_test(false, opts->uri, opts->preserve) == EXIT_SUCCESS);
-
-    testutil_assert(run_test(true, opts->uri, opts->preserve) == EXIT_SUCCESS);
+    for (i = 0; i < 2; ++i) {
+        for (j = 0; j < 2; ++j)
+            testutil_assert(run_test(i % 2, j % 2, opts->uri, opts->preserve) == EXIT_SUCCESS);
```

Comment:
Is the `%` operator necessary here?

Reply:
Nope, updated in 66f6d886dcd33e2e0956ff5bac86e0ab9be17ba9.